config/settings.yml

```diff
@@ -721,16 +671,9 @@
       - ShutdownGuest_Complete
       - SuspendVM_Task
```

Comment:
@durandom I think all CamelCased events are VMware related.  @Fryguy Any idea?
```SuspendVM_Task``` can be found [here](http://pubs.vmware.com/vsphere-65/index.jsp#com.vmware.wssdk.apiref.doc/vim.Task.html). 
There is another [VMware link](http://pubs.vmware.com/vsphere-65/index.jsp#com.vmware.wssdk.apiref.doc/vim.event.Event.html) about events.